packages/components/package.json

```diff
@@ -43,6 +43,7 @@
 		"@wordpress/is-shallow-equal": "file:../is-shallow-equal",
 		"@wordpress/keycodes": "file:../keycodes",
 		"@wordpress/primitives": "file:../primitives",
+		"@wordpress/react-i18n": "file:../react-i18n",
```

Comment:
I think, you still need to run `npm install` so it passes on CI. It uses `npm ci` which might work differently than `npm i` 🤷🏻